pkg/asset/rhcos/image.go

```diff
@@ -86,8 +85,10 @@ func osImage(config *types.InstallConfig) (string, error) {
 			return config.Platform.AWS.AMIID, nil
 		}
 		region := config.Platform.AWS.Region
-		if !configaws.IsKnownRegion(config.Platform.AWS.Region, config.ControlPlane.Architecture) {
-			region = "us-east-1"
+		if _, ok := streamArch.Images.Aws.Regions[region]; !ok {
```

Comment:
Can we find a way to re-use the code in the `rhcos.AMIRegions` function here as well?